package.json

```diff
@@ -3,3 +3,6 @@
   "private": true,
   "dependencies": {}
 }
+"scripts": {
+    "start": "node server.js"
+}
```

Comment:
What's this?

Reply:
This was put in on the advice of heroku help